packages/api/test/cache-custom.test.ts

```diff
@@ -24,9 +25,13 @@ describe('cache (custom directory)', function () {
     await fs.rmdir(cacheDir, { recursive: true });
 
     Cache.setCacheDir(originalCacheDir);
+
+    fetchMock.restore();
```

Comment:
Missed this somehow so if you can this test with `-w` and made changes it'd crash with an error saying that the mock it was trying to mock out in the test was already set up.